docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -2009,7 +2009,7 @@ Depending on the value of `sampler_type`, the sampler configuration parameter ca
 
 When `sampler_type` is `remote`, this specifies the URL of the sampling server. This can be used by all tracing providers.
 
-Use a sampling server that supports the Jaeger remote sampling API, such as jaeger-agent, jaeger-collector, opentelemetry-collector-contrib, or [Grafana Agent](/oss/agent/).
+Use a sampling server that supports the Jaeger remote sampling API, such as jaeger-agent, jaeger-collector, opentelemetry-collector-contrib, or [Grafana Alloy](https://grafana.com/oss/alloy-opentelemetry-collector/).
```

Comment:
Interesting link target for Alloy. It's not a bad link, but unexpected vs a link to the Alloy docs at https://grafana.com/docs/alloy/latest/

Reply:
I was just replacing like with like. Do you want a docs link there?

Reply:
I was replacing like with like. Do you want a docs link there?

Reply:
Either/or. It's not a bad link. It references the Alloy marketing info on the main site. Leave it.. merge it :-)